src/App.tsx

```diff
@@ -1,27 +1,49 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { GoodsList } from './GoodsList';
-
-// import { getAll, get5First, getRed } from './api/goods';
+import { Good } from './types/Good';
+import { getAll, get5First, getRedGoods } from './api/goods';
 // or
 // import * as goodsAPI from './api/goods';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState([] as Good[]);
+  /* eslint-disable */
+  function setVisibleGoods(callback: any) {
```

Comment:
Avoid using 'any' as a type declaration. It defeats the purpose of using TypeScript because it does not provide any type safety.